apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -29,36 +28,32 @@ const styles = {
  * A menu with a set of clickable links that calls the cancel handler if you
  * click outside the menu or the cancel button.
  */
-export const JavalabContextMenu = class JavalabContextMenuComponent extends Component {
+export const JavalabTabMenu = class JavalabTabMenuComponent extends Component {
```

Comment:
do we need to export this? Or do we only ever use the default export at the bottom?

Reply:
Oops, we don't need to export it, I'll remove that and the same thing in the file explorer